third_party/java/arcs/build_defs/internal/manifest.bzl

```diff
@@ -27,15 +31,10 @@ def arcs_manifest(name, srcs, deps = [], visibility = None):
         visibility = visibility,
     )
 
-    test_args = " ".join(["--src $(location %s)" % src for src in srcs])
-
-    sigh_command(
-        name = name + "_test",
-        srcs = all_files,
-        sigh_cmd = "run manifestChecker " + test_args,
-        deps = [name],
-        progress_message = "Checking Arcs manifest",
-        execute = False,
+    arcs_manifest_parse_test(
+        name = name + "_parse_test",
+        srcs = srcs,
+        deps = all_files,
```

Comment:
Can deps be just deps? Does it also need to include srcs?

Reply:
Good idea, done